activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -238,17 +245,19 @@ def define_acronym_regex_patterns
 
     # Yields a singleton instance of Inflector::Inflections so you can specify
     # additional inflector rules. If passed an optional locale, rules for other
-    # languages can be specified. If not specified, defaults to <tt>:en</tt>.
+    # languages can be specified. If not specified, defaults to I18n.default_locale.
     # Only rules for English are provided.
     #
     #   ActiveSupport::Inflector.inflections(:en) do |inflect|
     #     inflect.uncountable 'rails'
     #   end
-    def inflections(locale = :en)
+    def inflections(locale = nil)
+      locale ||= I18n.default_locale
```

Comment:
I don't think we can default to the `default_locale` as is. Rails use inflections a lot internally and we usually expect the `:en` locale. If the app change the default locale it should not change the inflections for generators for example. If people want inflections to use the `default_locale` I think they should pass the locale argument always.